scripts/measures/build-2018-measures

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+currentPerformanceYear=$1
```

Comment:
you can set this to =2018 now, no need to pass it in

also for consistency you can have this file be measures/2018/build-measures